ui/app/common/displaycontrols/forms/views/formsTable.html

```diff
@@ -22,13 +22,11 @@
                             <span ng-hide="::showFormsDate">{{data.obsDatetime || data.encounterDateTime | bahmniDateTime}}</span>
                             <span ng-show="::showFormsDate">{{
                                 data.obsDatetime || data.encounterDateTime | bahmniDate}}</span>
-                            <span ng-show="doesUserHaveAccessToTheForm(data ,'view')"class="has-link fl formDataEdit"
-                                  ng-dialog="../common/displaycontrols/forms/views/showObservationFormData.html"
-                                  ng-dialog-class="ngdialog-theme-default ng-dialog-all-details-page ng-dialog-view"
-                                  ng-dialog-data="{{getConfigToFetchDataAndShow(data)}}">
+                            <span ng-show="doesUserHaveAccessToTheForm(data, 'view')" class="has-link fl formDataEdit"
+                                  ng-click="openDialog(data)">
```

Comment:
@kavitha-sundararajan This refactoring is breaking the view functionality of old forms2 display control.

Reply:
Reverted this change.